pepper-apis/src/main/java/org/broadinstitute/ddp/pex/TreeWalkInterpreter.java

```diff
@@ -260,14 +260,9 @@ private boolean applyFormPredicate(InterpreterContext ictx, FormPredicateContext
                         }
                     })
                     .collect(Collectors.toList());
-
             return fetcher.findLatestActivityInstanceStatus(ictx, studyGuid, activityCode)
-                    .map(latestStatus -> expectedStatuses.contains(latestStatus))
-                    .orElseThrow(() -> {
-                        String msg = String.format("No activity instance of form %s found for user %s and study %s",
-                                activityCode, ictx.getUserGuid(), studyGuid);
-                        return new PexFetchException(new NoSuchElementException(msg));
-                    });
+                    .map(expectedStatuses::contains)
+                    .orElse(false);
```

Comment:
This change will reduce a lot of noise from https://console.cloud.google.com/errors/CNq4jMrM5eyIEA?time=P30D&project=broad-ddp-prod.